src/gui/mainwindow.ui

```diff
@@ -466,6 +467,11 @@
     <string>Critical Messages</string>
    </property>
   </action>
+  <action name="actionClose_Window">
```

Comment:
I think this should be camel case.

Reply:
This is autogenerated, but I am fine to change it if you insist.

Reply:
Coding guidelines also applies here, I suppose. So please change it.